Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
+VERSION := $(shell echo $(shell git describe --tags 2>/dev/null || git log -1 --format='%h') | sed 's/^v//')
```

Comment:
I've added `2>/dev/null` bc, in case there's no tag, git prints something like:
```
fatal: No names found, cannot describe anything.
```
and we don't need this message

Reply:
also mention:
`--format='%h'` -> short sha
`--format='%H'` -> long sha